test/commands/markdown/parse_test.rb

```diff
@@ -106,4 +103,12 @@ class Markdown::ParseTest < ActiveSupport::TestCase
   test "parses double tildes as strikethrough" do
     assert_equal "<p><del>Hello</del></p>\n", Markdown::Parse.("~~Hello~~")
   end
+
+  test "removes level one headings by default" do
+    assert_equal "<p>Content</p>\n", Markdown::Parse.("# Top heading\n\nContent")
+  end
+
+  test "can keep level one headings" do
+    assert_equal "<h1>Top heading</h1>\n<p>Content</p>\n", Markdown::Parse.("# Top heading\n\nContent", strip_h1: false) # rubocop:disable Naming/VariableNumber
+  end
```

Comment:
Nice. So if you add an extra test here to guard against when `# ...` is used in code, then you can TDD the solution.